llama_index/indices/query/response_synthesis.py

```diff
@@ -76,7 +112,9 @@ def from_args(
         optimizer: Optional[BaseTokenUsageOptimizer] = None,
         verbose: bool = False,
     ) -> "ResponseSynthesizer":
-        """Initialize response synthesizer from args.
```

Comment:
if we do land this, might want to swap implementation of this fn for using ResponseContext under the hood